dev-server/documents/html/z-index.mustache

```diff
@@ -0,0 +1,24 @@
+ <!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Hypothesis Client Test</title>
+<style>
+  body {
+    font-family: sans-serif;
+  }
+</style>
+</head>
+<body>
+  <h2>Test z-index</h2>
```

Comment:
On this test page there is an issue when selecting the title:

<img width="371" alt="Title z-index issue" src="https://user-images.githubusercontent.com/2458/95067505-d977d800-06fb-11eb-9c7e-e0903fa4d9dc.png">

This is because in this instance the location where the adder is shown has higher z-indexes than the locations for which the z-index is determined.

I don't know how common this will be in practice, but we might need to consider other ways of enumerating the elements to be checked.